era/base.js

```diff
@@ -398,70 +453,93 @@ export function handler_castCanMissNoCoefficient(threatValue) {
   return (ev, fight) => {
     let t = ev.type;
     if (t === "cast") {
-      threatFunctions.sourceThreatenTarget(ev, fight, threatValue, false);
+      threatFunctions.sourceThreatenTarget({
+        ev,
+        fight,
+        amount: threatValue,
+        useThreatCoeffs: false,
+      });
     } else if (t === "damage") {
-      threatFunctions.sourceThreatenTarget(ev, fight, -threatValue, false);
+      threatFunctions.sourceThreatenTarget({
+        ev,
+        fight,
+        amount: -threatValue,
+        useThreatCoeffs: false,
+      });
     }
   };
 }
 
 export function handler_modDamage(multiplier) {
   return (ev, fight) => {
     if (ev.type !== "damage") return;
-    threatFunctions.sourceThreatenTarget(
+    threatFunctions.sourceThreatenTarget({
       ev,
       fight,
-      ev.amount + (ev.absorbed || 0),
-      true,
-      multiplier
-    );
+      amount: ev.amount + (ev.absorbed || 0),
+      multiplier,
+      debugLabel: `modDamage`,
+    });
   };
 }
 export function handler_modHeal(multiplier) {
   return (ev, fight) => {
     if (ev.type !== "heal") return;
-    threatFunctions.unitThreatenEnemiesSplit(
+    threatFunctions.unitThreatenEnemiesSplit({
       ev,
-      "source",
+      unit: "source",
       fight,
-      (multiplier * ev.amount) / 2
-    );
+      amount: ev.amount,
+      multiplier: multiplier * 0.5,
+      debugLabel: `modHeal (${multiplier} * 0.5)`,
```

Comment:
In most cases I've added a debugLabel with the handler so that we can find the source